addons/sys_gestures/XEH_postInit.sqf

```diff
@@ -0,0 +1,70 @@
+#include "script_component.hpp"
+if (is3DEN || {!hasInterface || {!(GVAR(gesturesEnabled))}}) exitWith {};
+if (!isClass (configFile >> "CfgPatches" >> "ace_common")) exitWith {}; // No ACE exit
+
+GVAR(vestRadioArr) = call compile (GVAR(vestRadios));
+GVAR(headsetRadioArr) = call compile (GVAR(headsetRadios));
```

Comment:
Possibility for `parseSimpleArray`? Not sure I like compiling easily-inputted user strings like that.